org.kde.skrooge.json

```diff
@@ -30,13 +30,13 @@
     "modules": [
         {
             "name": "tcl",
+            "//": "sqlcipher requires tclsh during build",
             "subdir": "unix",
             "build-options": {
                 "no-debuginfo": true
             },
             "cleanup": [
-                "/bin",
-                "/man"
+                "/*"
```

Comment:
Can you explain this? Doesn't cleanup /* remove everything?

Reply:
Yes, but as I commented, tclsh is only needed during build. It's poorly documented, but looking at build output on binary-factory.kde , after `Committing stage build-skrooge to cache` the cleanup begins with `Cleaning up`. So everything only needed to build can be wiped out. Same goes for gengetopt below. I could be wrong, but it works for [kdeapps builds](https://binary-factory.kde.org/job/Skrooge_x86_64_flatpak/lastBuild/consoleText).